app/Jobs/Schedule/RunTaskJob.php

```diff
@@ -27,13 +27,19 @@ class RunTaskJob extends Job implements ShouldQueue
      */
     public $task;
 
+    /**
+     * @var bool
+     */
+    public $now;
+
     /**
      * RunTaskJob constructor.
      */
-    public function __construct(Task $task)
+    public function __construct(Task $task, $now = false)
     {
         $this->queue = config('pterodactyl.queues.standard');
         $this->task = $task;
+        $this->now = $now;
```

Comment:
This is a pretty confusing variable name from trying to parse the logic here. This can probably be renamed to something like `$runNow` or `$manualRun` might be even better since that seems to be what it actually is.